docs/source/conf.py

```diff
@@ -298,3 +298,17 @@
     'issue': ('https://github.com/dask/dask/issues/%s', 'GH#'),
     'pr': ('https://github.com/dask/dask/pull/%s', 'GH#')
 }
+
+#  --Options for sphinx extensions -----------------------------------------------
+
+intersphinx_mapping = {'pandas': ('http://pandas.pydata.org/pandas-docs/stable/',
+                                  'http://pandas.pydata.org/pandas-docs/stable/objects.inv'),
+                       'numpy': ('https://docs.scipy.org/doc/numpy/',
+                                 'https://docs.scipy.org/doc/numpy/objects.inv')}
+
+#  --Options for sphinx extensions -----------------------------------------------
+
+intersphinx_mapping = {'pandas': ('http://pandas.pydata.org/pandas-docs/stable/',
+                                  'http://pandas.pydata.org/pandas-docs/stable/objects.inv'),
+                       'numpy': ('https://docs.scipy.org/doc/numpy/',
+                                 'https://docs.scipy.org/doc/numpy/objects.inv')}
```

Comment:
Looks like this section has been duplicated here.

Reply:
Some old files have joined this PR.
I'm going to remove them and resubmit with the changes of only relevant for str.cat

Reply:
Ok sure. Is there a PR for the intersphinx stuff somewhere?

Reply:
Ah found it #3050. Yeah I assume this is a "squash and merge" issue.